core/state/statedb_utils.go

```diff
@@ -95,14 +100,15 @@ func IsAddressBlacklisted(statedb *StateDB, blacklistAddr *common.Address, addre
 	contract := *blacklistAddr
 	disabledSlot := slotBlacklistContractMapping[DISABLED]
 	disabled := statedb.GetState(contract, GetLocSimpleVariable(disabledSlot))
-	if disabled.Big().Cmp(big.NewInt(1)) == 0 {
+
+	if disabled == valueOne {
```

Comment:
I think we wanna change value to variable, then it should show its purpose. `disabledStateValue` using this name is better